source/content/guides/edge-integrations/01-introduction.md

```diff
@@ -32,16 +32,16 @@ The benefits of Edge Integrations:
 	- You can improve productivity by using existing content, style guides, media, and CMS integrations. No need to manage assets in multiple places.
 
 - Performance-forward
-	- Improve credibility with a fast, seamless customer experience- distribute personalized content across dozens of global and US points of presence.
+	- Improve credibility with a fast, seamless customer experience, and distribute personalized content across dozens of global and US points of presence.
```

Comment:
No comma before "and" because you are not listing three or more things, you're connecting two separate concepts.
```suggestion
	- Improve credibility with a fast, seamless customer experience and distribute personalized content across dozens of global and US points of presence.
```